client/components/contact-box/contact.js

```diff
@@ -39,27 +39,30 @@ export default class Contact extends React.Component {
             </div>
             <div className="owisp-contact-links">
               {social_links.map(link => {
-                return (
-                  <a
-                    href={link.url}
-                    target="_blank"
-                    rel="noopener noreferrer"
-                    key={link.url}
-                    className={`owisp-contact-${getText(
-                      link.alt,
-                      language,
-                    )}-link owisp-contact-link`}
-                  >
-                    <img
-                      src={getAssetPath(orgSlug, link.icon)}
-                      alt={getText(link.alt, language)}
+                if (link.authenticated === undefined || link.authenticated === isAuthenticated) {
```

Comment:
this if looks repeated across different components, may be worth to move it to a simple function stored in utils